### PR TITLE
arc-gtk-theme: init at f4c71247cf9470037d052ae4a12b86073d0001ff

### PR DESCRIPTION
**Pull request by simonvandel**

### CODE REVIEW

File: pkgs/misc/themes/arc/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, autoconf, automake, fetchFromGitHub, gnome3, gtk, gtk-engine-murrine, pkgconfig}:
+
+stdenv.mkDerivation rec {
+  version = "f4c71247cf9470037d052ae4a12b86073d0001ff";
```
Comment (jagajaga) [inline pkgs/misc/themes/arc/default.nix]: Please fix version according to https://nixos.org/nixpkgs/manual/#sec-package-naming.

**Comment by simonvandel** [comment]
@jagajaga I made some changes

**Comment by jagajaga** [comment]
Please squash commits into one.

**Comment by simonvandel** [comment]
@jagajaga

**Pull request merged**